leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,165 @@
+\subsection{Scoring}
+After all attempts, the best score for each target will be added together and recorded in the leaderboard.
+Distances are rounded to the closest centimetre.
+
+\section{Most Passes Leaderboard}
+This leaderboard measures the highest number of successful passes a team can
+complete in a given timeframe, emphasizing passing accuracy, speed, and coordination.
+
+\subsection{Setup}
+This leaderboard challenge will take place on a standard SPL field and requires 2 competing robots,
+a Game Controller and 1 standard SPL soccer ball.
+
+Competing robots will be placed on the touchlines on opposite sides of the same half of the field,
+in line with the penalty mark. The ball will be placed on either corner of the penalty area.
+
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/passing_leaderboard.pdf}}
+    \caption{Passing Kick Leaderboard: Robots completing the attempt are shown in blue, target area is shown in green, exclusion area is shown in magents and ball placement is shown in red}
+    \label{fig:passing_leaderboard}
+\end{figure}
+\subsection{Challenge Execution}
+In each attempt, a playing packet will be sent from the game controller to indicate the start of the attempt,
+and will begin counting down the timer.
+Teams will then have at most \qty{\MaxTimePassingLeaderboard}{\sec} to move the ball from the starting
+position to the target area while completing as many passes as possible. The centre
+1.5 metres of the field (Centre Cricle Diameter) is an exclusion zone, no robots are allowed within this area,
+if a robot enters this area, or the ball comes to a complete stop
+within this area, the run is ended immediately. If the ball is kicked off the field,
+it is placed back on the touchline where it crossed.
+
+The ref will announce each time the ball crosses the exlusion zone, and whether this will
+count as a successful pass. This is then captured by the Game Controller.
+
+Once a robot has posession of the ball within the target area, the ref will announce that the
+attempt has finished, then the finish signal will be sent from the game controller and the timer will stop.
+
+
+\subsection{Scoring}
+1 point is awarded every time the ball crosses the exlusion zone,
+with an additional point awarded for a "successful pass". A "successful pass" is defined as when the
+The ball stops in an 180◦ arc with radius 75cm in front of the receiving robot.
+
+No points are counted if the time runs out before a robot has posession of the ball
+within the target area, the ball hasn't crossed the exclusion zone at least once
+or if the run ends prematurely as per the above rules.
+
+The score is then multiplied by the number of seconds remaining on the game controller timer.
+
+The highest score of all attempts will be recorded in the leaderboard.
+
+\section{Ball Control Leaderboard}
+This leaderboard will measure how well a NAO can keep control of a ball, while moving it
```

Comment:
As this one share few thing with the walk leaderboard, maybe we should move this leaderboard right after the walk leaderboard ?